### DISCUSSION TITLE
LNAVMAP crashed twice today - please enlighten me what I did wrong :) - logfiles attached

### DISCUSSION THREAD

**Discussion by BattleOverlord** [albar965/littlenavmap#1204] [Q&A]
I was doing 7-8 hour flight from KBFI-BIKF (delivering "new" 737-800 from Renton to Keflavik). LNAVMAP crashed twice (I think it was already after I built new scenery library). Only difference should be that I've added two airports (KBFI and BIKF) before I started LNAVMAP and built the scenery library in LNAVMAP again. LNAVMAP almost never crashed for me so thats new and I would like to know what I did wrong.  Based on my limited knowledge I think it could cause crash, because I loaded simbrief FP and after that rebuilt the scenery? I dont know :)
[little_navmap_crashreport.zip](https://github.com/user-attachments/files/17684126/little_navmap_crashreport.zip)
[little_navmap_crashreport_1.zip](https://github.com/user-attachments/files/17684127/little_navmap_crashreport_1.zip)

Yes I use 789 performance file - I know.

Thanks!

**Comment by albar965** [comment]
Thanks for the reports but unfortunately the stack trace is incomplete. Can happen in some cases.
You can also try the  [3.0.12](https://albar965.github.io/release/2024/11/03/littlenavmap-stable-3012-released.html).

I'll check again with your settings.

**Reply by albar965** [reply]
Tried with your settings and no crash. I had a few ones (fixed) using the Web Server but there are maybe still some gremlins in there. Disable the Web Server if you do not use it.

**Reply by BattleOverlord** [reply]
Ok turned it off. Thank you.
